src/overlays/actors/ovl_Bg_Ikana_Mirror/z_bg_ikana_mirror.c

```diff
@@ -4,7 +4,10 @@
  * Description: Stone Tower Temple - Mirror
  */
```

Comment:
If you want you can add a general note on the behaviour of this actor (see z_en_holl.c for an extreme example). I imagine most people reading this file are only going to want to know about the charging and emission, so you might as well just describe that here.